### ISSUE TITLE
DataTable warnings with MSVC.

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-core#692]
```
C:\projects\opensim-core\OpenSim/Common/DataTable.h(368): warning C4267: '=': conversion from 'size_t' to 'unsigned int', possible loss of data [C:\projects\opensim-core\build\OpenSim\Examples\DataTable\example1DataTable.vcxproj]
C:\projects\opensim-core\OpenSim/Common/DataTable.h(368): warning C4267: '=': conversion from 'size_t' to 'unsigned int', possible loss of data [C:\projects\opensim-core\build\OpenSim\Examples\DataTable\example2DataTable.vcxproj]
```

There are additional similar warnings as well.

@klshrinidhi 

Thanks @aymanhab

**Comment by sherm1** [comment]
I think I fixed this in PR #736 -- Chris, can you verify and close if so?

**Comment by chrisdembia** [comment]
Thanks.
